packages/console/src/command-history/CommandHistory.tsx

```diff
@@ -32,14 +32,17 @@ import { ItemAction, HistoryAction } from './CommandHistoryTypes';
 
 const log = Log.module('CommandHistory');
 
-type Settings = {
+export type CommandHistorySettings = {
   value: string;
-  language: string;
+  language?: string;
 };
 interface CommandHistoryProps {
-  language: string;
+  language?: string;
```

Comment:
`language` should still be required in `@deephaven/console`, I think you changed it to optional in a few places. `CommandHistoryPanel` shouldn't be rendering `CommandHistory` without a language.